hydradx/model/model_initialization.py

```diff
@@ -1,49 +1,50 @@
 import pandas as pd
 
-def initialize_model(initial_liquidity, initial_tradevolume, initial_fee_assets, initial_fee_HDX):
 
-########## AGENT CONFIGURATION ##########
-# key -> token name, value -> token amount owned by agent
-# note that token name of 'omniABC' is used for omnipool LP shares of token 'ABC'
-# omniHDXABC is HDX shares dedicated to pool of token ABC
+def initialize_model(initial_liquidity, initial_tradevolume, initial_fee_assets, initial_fee_LRNA):
+
+    ########## AGENT CONFIGURATION ##########
+    # key -> token name, value -> token amount owned by agent
+    # note that token name of 'omniABC' is used for omnipool LP shares of token 'ABC'
+    # omniHDXABC is HDX shares dedicated to pool of token ABC
```

Comment:
We no longer have HDX or LRNA shares ownable by LPs, this line can be removed so as to avoid confusion